validator-api/src/contract_cache/mod.rs

```diff
@@ -25,12 +27,16 @@ use tokio::time;
 use validator_api_requests::models::{MixNodeBondAnnotated, MixnodeStatus};
 use validator_client::nymd::CosmWasmClient;
 
+pub(crate) mod reward_estimate;
 pub(crate) mod routes;
 
 pub struct ValidatorCacheRefresher<C> {
     nymd_client: Client<C>,
     cache: ValidatorCache,
     caching_interval: Duration,
+
+    // Readonly: some of the quantities cached depends on values from the storage.
+    storage: Option<ValidatorApiStorage>,
```

Comment:
personally I dislike putting it here as it doesn't really fit in here... but oh well...

Reply:
Hot take: it's not a contract cache (anymore) :wink: